src/ScrollContainer/index.tsx

```diff
@@ -19,25 +21,16 @@ const ScrollContainer: React.FunctionComponent<IScrollContainer> = ({
   autosize = true,
   ...props
 }) => {
-  const [dragEndTimeout, setDragEndTimeout] = React.useState<number | NodeJS.Timer | null>(null);
-
   const scrollbar = React.useRef<any>(null);
 
-  const handleDragEnd = React.useCallback(() => {
-    setDragEndTimeout(window.setTimeout(hideTracks, 500));
-  }, [dragEndTimeout]);
-
   const showTracks = React.useCallback(() => {
     if (!scrollbar.current) return;
 
     scrollbar.current.trackXElement.style.opacity = 1;
     scrollbar.current.trackYElement.style.opacity = 1;
-
-    if (!dragEndTimeout) return;
-
-    window.clearTimeout(dragEndTimeout as number);
-    setDragEndTimeout(null);
-  }, [scrollbar, dragEndTimeout]);
+    scrollbar.current.trackXElement.style.transition = 'opacity 0s';
```

Comment:
This could be set permanently.

Reply:
It'd be nice to have some transition when you hover over, so perhaps keep it but just change the value.

Reply:
But then it would be permanently overridden by `opacity 1s` from `hideTracks`, wouldn't it?
Other solution could be setting classes for those two events.

Reply:
Yeah, you can keep it as is yet it would be nice to add a small transition here.